packages/integration-karma/test/component/LightningElement.errorCallback/index.spec.js

```diff
@@ -8,13 +8,26 @@ import XBoundaryChildSlotThrow from 'x/boundaryChildSlotThrow';
 import XNestedBoundaryChildThrow from 'x/nestedBoundaryChildThrow';
 import XBoundaryChildSelfRehydrateThrow from 'x/boundaryChildSelfRehydrateThrow';
 import XBoundaryAlternativeViewThrow from 'x/boundaryAlternativeViewThrow';
+import XBoundaryRenderedThrowFrozen from 'x/boundaryChildRenderedThrowFrozen';
 
 import XChildConstructorThrowDuringInit from 'x/childConstructorThrowDuringInit';
 import XChildRenderThrowDuringInit from 'x/childRenderThrowDuringInit';
 import XChildRenderedThrowDuringInit from 'x/childRenderedThrowDuringInit';
 import XChildConnectedThrowDuringInit from 'x/childConnectedThrowDuringInit';
 
 describe('error boundary', () => {
+    it('should not throw when the error object is frozen', () => {
+        const elm = createElement('x-boundary-rendered-throw-frozen', {
+            is: XBoundaryRenderedThrowFrozen,
+        });
+        document.body.appendChild(elm);
+
+        return Promise.resolve().then(() => {
+            const alternativeView = elm.shadowRoot.querySelector('.rendered-frozen-alternative');
+            expect(alternativeView.textContent).toEqual('rendered frozen alternative view');
```

Comment:
Instead of checking the presence of the alternative view, I would rather check the error itself. And see if all `wcStack` is set of not.